feg/gateway/sbi/sbi_test.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sbi_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+
+	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	n7 "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControlServer"
+	sbi "magma/feg/gateway/sbi/specs/TS29571CommonData"
+)
+
+type MockPcf struct {
```

Comment:
maybe it would be better to have the echo server exposed here instead of `ListenAddr`. This way you can access any propery. Note this is just a mock object